lib/BloqadeQMC/test/rydberg.jl

```diff
@@ -1,15 +1,16 @@
-using Test, BloqadeQMC
+using Test
 using Statistics
 using Random
 using RandomNumbers
 using Measurements
 using Measurements: value, uncertainty
-using BinningAnalysis
-using BloqadeQMC: Chain, Square
 using BloqadeLattices: generate_sites, ChainLattice
+using BloqadeQMC: rydberg_QMC, BinaryThermalState, Diagnostics, mc_step_beta!
```

Comment:
consider export `rydberg_QMC`?

Reply:
Added an export statement but again we need to look more closely into the export / import / using statements overall. Weird things break, so I'm leaving this line for now :)